lib/openstack/openstack_event_monitor.rb

```diff
@@ -25,7 +28,7 @@ def self.plugin_priority
   # ordering of plugins.
   def self.subclasses
     # sort plugins on plugin_priorty
-    super.sort_by(&:plugin_priority)
+    (super - [OpenstackNullEventMonitor]).sort_by(&:plugin_priority)
```

Comment:
I don't understand the need to remove this from the subclasses.  And since subclasses is a Ruby method, I'm concerned about removing anything from it.  The old code just ordered it...this actually changes it.